test/puppetlabs/puppetdb/command_test.clj

```diff
@@ -866,7 +866,7 @@
 
 (deftest replace-facts-bad-payload
   (let [bad-command {:command (command-names :replace-facts)
-                     :version 5
+                     :version (apply max (get supported-command-versions "replace facts"))
```

Comment:
@mwaggett this change seems fine to me at a high level, but it'd be cleaner if these apply max things could just be swapped out for references to that command-versions map you have in export.clj .

If you go that route I'd also think about moving that map to command.clj where supported-command-versions lives, since at that point it'll be more general than just export.